larray/inout/xw_reporting.py

```diff
@@ -0,0 +1,709 @@
+import os
+import warnings
+from collections import OrderedDict
+
+from larray.util.misc import PY2, _positive_integer
+from larray.core.group import _translate_sheet_name
+from larray.core.array import aslarray
+from larray.example import load_example_data, get_example_templates_dir
+
+try:
+    import xlwings as xw
+except ImportError:
+    xw = None
+
+
+_default_items_size = {}
+
+
+def _validate_template_dir(template_dir):
```

Comment:
Since the code is generic, maybe rename this to _validate_dir or _check_is_dir and change the error message?